Tests/IntegrationTests.Shared/InstanceTests.cs

```diff
@@ -62,6 +62,16 @@ public void GetInstanceWithJustFilenameTest()
             // Arrange, act and "assert" that no exception is thrown, using default location + unique name
             Realm.GetInstance(specialRealmName).Close();
         }
+        
+        [Test]
```

Comment:
Might we not just as well move this (and the `ObjectIntegrationTests.CreateObjectFromPCLTest` test below) into the PCL themselves? Seems to me that they belong in there.